src/python/director/motionplanningpanel.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, robotStateModel, robotStateJointController, teleopRobotModel,
         # Foot step planning
         self.placer = None
         self.ui.walkingPlanButton.connect('clicked()', self.onWalkingPlan)
-        
+        self.ui.walkingFakeExeButton.connect('clicked()', self.onWalkingFakeExecute)
```

Comment:
'fake exe' simply move the robot to the desired location without actual 'walking'. It provides a way to evaluate and test final pose planning and motion planning in a much simpler manner